inc/saiswitch.h

```diff
@@ -2776,6 +2776,29 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_ECMP_MEMBER_COUNT,
 
+    /**
+     * @brief Enable or disable credit watchdog
+     *
+     * Credit Watchdog can be enabled or disabled using this attribute for VOQ based system
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     * @validonly SAI_SWITCH_ATTR_TYPE == SAI_SWITCH_TYPE_VOQ
+     */
+    SAI_SWITCH_ATTR_CREDIT_WD,
```

Comment:
currently without this in sai, should watch-dog be already enabled by SDK/in hardware already? Should defualt be true, and application no need to enable it?

Reply:
Yes, checked with deployments, WD default should be true.
https://github.com/opencomputeproject/SAI/pull/1748/commits/bcc016276ebe9d00d9d8aae25caf48e3ae310b69